### ISSUE TITLE
Ckeditor 4.6.2 : Adding class in chrome instead of inline style in chrome

### ISSUE THREAD

**Issue by ratneshoza** [ckeditor/ckeditor-dev#814]
Can you help me to understand  while copy paste content from word to inline ckeditor are adding class instead of inline style in chrome. For example:

Copy content from word in chrome :

```html
<p class="FundHeader">Global Income</p>
```

Actual content in word :

```html
<p style="margin-left:0in; margin-right:0in"><span style="font-size:26pt"><span style="font-family:Arial,sans-serif"><span style="color:#006395">Global Income</span></span></span></p>
```

**Comment by mlewand** [comment]
Updated original comment formatting.

**Comment by mlewand** [comment]
It does look to me as if it uses raw datatransfer.

@ratneshoza can you check the same in CKEditor 4.7.2? For instance http://cdn.ckeditor.com/4.7.2/full-all/samples/

In release 4.7.x we implemented style inlining for content pasted from MS Word.

**Comment by ratneshoza** [comment]
Thanks mlewand for quick response i have just updated the comment.

**Comment by mlewand** [comment]
I'm sorry @ratneshoza we can't provide patches for previous releases. Eventually you could check if it got fixed in CKEditor 4.7.x and backport it on your own, but that would be rather a challenging task - I'd recommend updating.
